server/controller/statistic.js

```diff
@@ -0,0 +1,30 @@
+const sequelize = require('../database/config');
+const { questions, users, translations } = require('../database/models');
+
+exports.get = async (request, response) => {
+  try {
+    const getUsersCount = await users.findAll({
+      attributes: [[sequelize.fn('COUNT', sequelize.col('id')), 'usersCount']],
+    });
+    const getQuestionsCount = await questions.findAll({
+      attributes: [[sequelize.fn('COUNT', sequelize.col('id')), 'questionsCount']],
+    });
+    const getTranslationsCount = await translations.findAll({
+      attributes: [[sequelize.fn('COUNT', sequelize.col('id')), 'translationsCount']],
+    });
+
+    const usersCount = getUsersCount[0].dataValues;
+    const questionsCount = getQuestionsCount[0].dataValues;
+    const translationsCount = getTranslationsCount[0].dataValues;
+
+    Promise.all([usersCount, questionsCount, translationsCount]).then((res) => {
+      if (!res.length) {
+        response.status(404).send('Oops! No data! ');
+      } else {
+        response.status(200).send(res);
```

Comment:
@ali-7 If this endpoint could return an object instead of an array, I think that would make it easier for a client to use and easier to add new counts in the future.
I think right now it responds like this:
`[{usersCount: 123}, {questionsCount: 456}, {translationsCount: 789}]`
So the client has to know to look at the first index for users, the second for questions, and so on.  If it was a flatter object:
`{usersCount: 123, questionsCount: 456, translationsCount: 789}` the client wouldn't have to have that extra knowledge about the order of the array.
And then you could add a new item count to the response without having to change the client code at all.  If you still had the array response and you added "adminsCount" to the beginning of the Promise.all call, then clients would have to update the index that they look for users, questions, translations at.